### ISSUE TITLE
Uncaught TypeError: Cannot read property 'add' of undefined

### ISSUE THREAD

**Issue by hwschuur** [atom-haskell/ide-haskell#190]
[Enter steps to reproduce:]

1. Wait until '*Main>' is shown in REPL window
2. Type some text and press enter

**Atom**: 1.14.0-beta3 ia32
**Electron**: 1.3.13
**OS**: Microsoft Windows 10 Enterprise
**Thrown From**: [ide-haskell](https://github.com/atom-haskell/ide-haskell) package 1.9.1

### Stack Trace

Uncaught TypeError: Cannot read property 'add' of undefined

```
At C:\Users\HSR\.atom\packages\ide-haskell\lib\editor-control.coffee:106

TypeError: Cannot read property 'add' of undefined
    at EditorControl.module.exports.EditorControl.markerFromCheckResult (/packages/ide-haskell/lib/editor-control.coffee:106:23)
    at EditorControl.module.exports.EditorControl.updateResults (/packages/ide-haskell/lib/editor-control.coffee:92:6)
    at EditorControl.updateResults (/packages/ide-haskell/lib/editor-control.coffee:1:1)
    at PluginManager.module.exports.PluginManager.updateEditorsWithResults (/packages/ide-haskell/lib/plugin-manager.coffee:57:24)
    at /packages/ide-haskell/lib/plugin-manager.coffee:17:54
    at Function.module.exports.Emitter.simpleDispatch (/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at ResultsDB.module.exports.ResultsDB.setResults (/packages/ide-haskell/lib/results-db.coffee:32:14)
    at UPIInstance.setMessages (/packages/ide-haskell/lib/upi.coffee:78:33)
    at IdeHaskellReplView.module.exports.IdeHaskellReplView.setError (/packages/ide-haskell-repl/lib/ide-haskell-repl-view.coffee:217:12)
    at ghci.GHCI.onError (/packages/ide-haskell-repl/lib/ide-haskell-repl-view.coffee:155:10)
    at Function.module.exports.Emitter.simpleDispatch (/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/app.asar/node_modules/event-kit/lib/emitter.js:129:28)
    at /packages/ide-haskell-repl/lib/ghci.coffee:85:20
    at /packages/ide-haskell-repl/lib/ghci.coffee:59:13)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    at Pipe.onread (net.js:543:20)
```

### Commands

```
     -1:31.4.0 ide-haskell-repl:toggle (input.hidden-input)
  3x -1:22 editor:newline (input.hidden-input)
  5x -1:20.5.0 core:backspace (input.hidden-input)
```

### Non-Core Packages

```
autocomplete-bibtex 0.7.0 
autocomplete-haskell 0.7.1 
haskell-ghc-mod 1.19.6 
ide-haskell 1.9.1 
ide-haskell-cabal 1.7.4 
ide-haskell-repl 0.5.1 
language-haskell 1.10.6 
language-latex 1.0.0 
latex 0.42.3 
latex-autocomplete 1.0.0 
latexer 0.3.0 
pdf-view 0.54.0 
sync-settings 0.8.1 
```

**Comment by lierdakil** [comment]
Well, Atom broke some API. **Again.** Everything should still work in 1.13. Thanks for letting me know, I'll see what I can do.

**Comment by lierdakil** [comment]
Hopefully fixed with v1.9.2. If not, please let me know.
